### ISSUE TITLE
JAWS not reading out underline formatting

### ISSUE THREAD

**Issue by JulesAchiel** [ckeditor/ckeditor-dev#432]
## Are you reporting a feature or a bug?
Bug

## [Check if the issue is already reported](http://docs.ckeditor.com/#!/guide/dev_issues_readme-section-avoid-duplicates)
none found

## Provide detailed reproduction steps (if any)
1. go to http://sdk.ckeditor.com/samples/fullpreset.html
2. go into source mode
3. Make the first line of text : `<u><b>brown</b></u> <i>fox</i>&nbsp;<u>jumps</u> of the <b><i>lazy</i></b> river`
4. exit source mode
5. start screen reader JAWS
6. position cursor in every single formatted word and press INSERT + F or INSERT + double F (quickly) to open up the virtual viewer
7. Notice that the underline formatting is never read out by JAWS

### Expected result
JAWS to read out the underline formatting

### Actual result
JAWS does not read out underline formatting

## Other details

* Browser: Chrome
* OS: Windows 7
* CKEditor version: 4.6.2
* Installed CKEditor plugins: full preset + wordcount,notification,preview,justify,placeholder_select,confighelper,balloonpanel,a11ychecker,autogrow,uploadimage,uploadwidget,widget,lineutils,clipboard,dialog,dialogui,widgetselection,filetools,notificationaggregator

**Comment by mlewand** [comment]
I can confirm this behavior in Chrome, however it's working fine in Firefox.

Also there's a buggy behavior on a regular `contenteditable` element, which shows that this is a clearly upstream, Chrome <-> JAWS bug.

Please report this issue to a relevant vendor. To be honest I'd report to both of them because I don't know which one has the problem in this case.

**Comment by mlewand** [comment]
@JulesAchiel still we'd be thankful for dropping a link here where it can be tracked when reported to a proper vendor.

**Comment by JulesAchiel** [comment]
There is no link to the issue with JAWS. JAWS support is done through email. Will keep this updated when I get more details.

**Comment by mlewand** [comment]
@JulesAchiel Great, thanks for the update. We'll be happy to hear back from you in case the issue has been fixed. Thanks!
